website/views/shared/profiles/achievements.jade

```diff
@@ -164,21 +164,21 @@ div(ng-if='profile.achievements.quests || user._id == profile._id')
         td x{{v}}
   hr
 
-div(ng-if='profile.achievements.snowball')
+div(ng-if='::profile.achievements.snowball')
   .achievement.achievement-snowball
   h5=env.t('annoyingFriends')
   small
     =env.t('annoyingFriendsText', {snowballs: "{{profile.achievements.snowball}}"})
```

Comment:
Probably? But I'm not absolutely sure, so I'll just merge it in as is now and look at the rest in more depth later. Not going to hurt anything to be slightly less optimized for a little while longer :smile: